crates/sui-core/src/signature_verifier.rs

```diff
@@ -394,8 +411,8 @@ pub fn batch_verify_certificates(
     certs: &[CertifiedTransaction],
 ) -> Vec<SuiResult> {
     // certs.data() is assumed to be verified already by the caller.
-
-    let verify_params = VerifyParams::new(Default::default());
+    // todo(joyqvq): check if this is safe to use default
```

Comment:
this is ok (at least for now) - the user sigs have already been verified. we should actually change the call below to `verify_committee_sigs_only` and then we can delete `verify_params`